integration-cli/docker_cli_run_test.go

```diff
@@ -450,12 +450,9 @@ func (s *DockerSuite) TestRunExitCode(c *check.C) {
 		exit int
 		err  error
 	)
-	if daemonPlatform == "windows" {
-		// FIXME Windows: Work out the bug in busybox why exit doesn't set the exit code.
-		_, exit, err = dockerCmdWithError("run", WindowsBaseImage, "cmd", "/s", "/c", "exit 72")
```

Comment:
b/c it turns out the issue was with the command parsing, so this can just use the same container spec for either platform.